script/Ciel/Day 10/10_01_CIEL10_2_2.txt

```diff
@@ -1,20 +1,27 @@
 [sha:c5555f2078564ddcf9b86dc21baafedd302cfa2f]{
 -- Page 10010, Offset 27391.
 -- 「……文句なんかありません。
-"I'm not complaining or anything.
+// 文句 comes up in offset 27379, but for EN flow it makes more sense to tie
+// it back to offset 27375, which is really more the topic at hand.
+"I don't hate you at all.
 }
 [sha:a627363983ade22465439c7968043722e6346645]{
 -- Page 10010, Offset 27392.
 -- だって、シエル先輩はそこにいてくれるだけで可愛いんですから。いつもぷにぷにしていて、それでいて健康的で、眼鏡が似合っていて、たまにイタズラなところもあって、色々な意味で目が離せない、最高の先輩です。
-I mean, you're cute just sitting around here, Ciel-senpai. You're the best Senpai I could ask for, always so soft and squishy, yet fit, looking good with glasses, sometimes a little teasing. I can't get my eyes off you, for many reasons.
+// Some liberties taken with そこにいてくれるだけで
+I mean, just look at you. You're adorable.
```

Comment:
Hmm, all of the things he goes on to list are purely visual (non-active) things, so I kinda prefer the more ogling version

Reply:
It just sounds a bit weird to me. 
"You're cute in just about every way there is" or "Everything about you is cute" seems like better options, in hindsight.

Reply:
I mean, that just makes it sound even more ill-fitting to me...

Reply:
It just doesn't have the same refuting oomph, imo.

Reply:
I can see the case for "you're cute no matter what you do" since that seems to literally match the source more, but "you're cute no matter what you do!" is more like "sure what you're doing might not be cute, but you are" and "you're cute in just about every way there is" doesn't fit closer to the text and is less natural than Ross's version.

"Everything about you is cute" is _decent_, but "I mean, just look at you. You're adorable" really is very natural, I'm not really sure why Ross used the grandma example (Ciel's glasses perhaps?) but the phrasing of like "I mean, just look at you. [evident thing]. [lists evidence]" fits really well imo and that ver. has a very "i need to walk someone off an argumentative ledge" urgency that "everything about you is cute" doesn't, if that makes sense.

Reply:
Consider me out-voted, but it doesn't sound like something he'd say.